cli/src/commands/util.rs

```diff
@@ -108,16 +119,21 @@ fn cmd_util_completion(
 fn cmd_util_gc(
     ui: &mut Ui,
     command: &CommandHelper,
-    _args: &UtilGcArgs,
+    args: &UtilGcArgs,
 ) -> Result<(), CommandError> {
     if command.global_args().at_operation != "@" {
         return Err(user_error(
             "Cannot garbage collect from a non-head operation",
         ));
     }
+    let keep_newer = SystemTime::now()
+        - match args.prune.as_deref() {
+            None => Duration::from_secs(14 * 86400),
+            Some("now") => Duration::ZERO,
+            _ => return Err(user_error("--prune only accepts 'now'")),
+        };
```

Comment:
nit: I wouldn't inline the match in math expression, but I'm not strongly against that.

(Perhaps, `SystemTime::now() - ` will be moved into the match if we add support for arbitrary time syntax including absolute one.)

Reply:
OK, I'll make it so that the match is not inlined in the subtraction.